### ISSUE TITLE
expand description for field "elevation"?

### ISSUE THREAD

**Issue by magelm** [NFDI4Microbiota/MetadataStandards#28]
Hey, 

IMO the description is quite short for the field "elevation" (e.g. in [animal-associated](https://github.com/NFDI4Microbiota/MetadataStandards/blob/main/Biological_Environmental/AnimalAssoc_BioEnv_Metadata.md) but prob all other tables as well ) and I cannot understand it as is without consulting other sources (like ENA or the mixs or google).

## Comparing the definitions

1. N4M: 

> Elevation is mainly used when referring to points on the earth’s surface. Origin elevation in m

2. MIxS: 

> Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit. [MIXS:0000093](https://w3id.org/mixs/0000093)

3. ENA: 

> The elevation of the sampling site as measured by the vertical distance from mean sea level. 

**Personal preference** is MixS as it is more descriptive and I would *almost* know how to measure this.

## **Solutions could be to:**
1. expand the description by adding more text, aka copy-pasting the definition either from ENA or Mixs (if not already identical)
2. to use a default description (cite) and add a separate column for the specific source of this description by add the URI & hyperlink to the/ or one full definition, e.g.: [MIXS:0000093](https://w3id.org/mixs/0000093)

currently, we have 5 rows:
![image](https://github.com/NFDI4Microbiota/MetadataStandards/assets/101332248/57b4f456-ff89-4e06-89f3-8f702635195f)

we would need at least one more row: 
![image](https://github.com/NFDI4Microbiota/MetadataStandards/assets/101332248/079c55b4-3e5e-49f4-9d4e-9ee46cb9031d)

and we should think about if recommended or expected "unit" should be a seperate column as well (in red; BUT this would be a different issue)

**Comment by magelm** [comment]
*Add-on for a more existential question that needs to be raised (please feel free to make this a separate issue):*

IMO, we need a bigger issue/ discussion in general as solving this issue #28 comes with restructuring in multiple tables/ levels. The discussion should then be about what our main goal is with this repository and the metadata tables specifically:

Are **tables** supposed to be a `reference` material or an `explanation?` If we can make this clear to us and the users, I think this will help tremendously. ATM our tables are in a weird limbo state between both.

PS: For the distinction, between the terms reference/ explanation, I would like to rely on the [definition and destinction from Diátaxis](https://diataxis.fr/reference-explanation/) and how we can quickly determine what our tables could/ should be:

![image](https://github.com/NFDI4Microbiota/MetadataStandards/assets/101332248/46d47137-b5db-4c9f-b4ec-709d66d352d3)

**Comment by magelm** [comment]
Aaaand more importantly: Elevation should neither be human nor animal-associated, right? Because this would be `height`, or not?

**Comment by MartinBoleSlo** [comment]
To address some of the questions

Comment 1: I wanted to keep it short and sweet, but if that misses the whole point of being understandable, then I agree. An expanded definition can be added to the table (and yes, descriptions should be the same in every table for the same metadata field). But the definition was sourced by MIxS, so I can reuse it, if that is the preferred option.

The columns can also be further split (as actually, that is a great idea), but I would require some time to do that.

Comment 2: Quote from page introduction "The primary objective of this GitHub page is to serve as a centralized repository for existing (meta)data standards. The purpose is to provide the international microbiological community with a comprehensive and easily accessible compilation of established standards, facilitating efficient navigation and utilization for researchers involved in collecting and submitting (meta)data to public repositories."  But I do agree, that we need to better define the purpose of the tables, but as to per my current knowledge, their purpose is actually both, but more on the explanation side (so row 4 from the definition and distinction from Diátaxis

Comment 3: I would personally keep the elevation, as to signify, if a sample was taken, for example, at 3000 m above sea-level or 200 m. Maybe for humans, the diet would probably not be significantly different, but let's say for animals, different diets (e.g. what plants they eat, which differ with elevation) can influence their microbiota. Please let me know if you think this is to much of a stretch and extrapolation in my thought process.

But altitude, I see the point, as the definition from MIxS reads: "Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air"

**Comment by magelm** [comment]
Thank you for your extensive replies and the explanation in "Comment 2"! 
I'll be replying to your comments one by one.

> Comment 2: [...] we need to better define the purpose of the tables, but as to per my current knowledge, their purpose is actually both, but more on the explanation side

If explanation, then a longer description for the terms might be better and more comprehensible, especially for newbies.


> Comment 1: [...] But the definition was sourced by MIxS, so I can reuse it, if that is the preferred option.

I'll follow the majority vote/ group consensus wether a short or longer description is preferred :) 
In any case, I would be in favor of adding the URI of the source description/ term, especially if MIxS was your reference point.

**Comment by magelm** [comment]
> Comment 3: I would personally keep the elevation, as to signify, if a sample was taken, for example, at 3000 m above sea-level or 200 m. Maybe for humans, the diet would probably not be significantly different, but let's say for animals, different diets (e.g. what plants they eat, which differ with elevation) can influence their microbiota. Please let me know if you think this is to much of a stretch and extrapolation in my thought process.

Agreed to keep it. In hindsight, I also noticed that elevation and altitudes are related to "site metadata" and not the host per se. My bad :D 

> But altitude, I see the point, as the definition from MIxS reads: "Heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air"

Also agree with this definition.
